### ISSUE TITLE
Problem with < for PBR semigroups

### ISSUE THREAD

**Issue by wilfwilson** [gap-packages/Semigroups#561]
It looks like `<` isn't implemented for PBR semigroups:
```
gap> A := Monoid([
>  PBR([[]], [[1]]),
>  PBR([[-1, 1]], [[1]]),
>  PBR([[-1]], [[]]),
>  PBR([[-1]], [[-1, 1]])]);
<pbr monoid of degree 1 with 4 generators>
gap> B := Monoid([
>  PBR([[]], [[]]),
>  PBR([[1]], [[]]),
>  PBR([[1]], [[-1]]),
>  PBR([[-1, 1]], [[1]]),
>  PBR([[-1]], [[]]),
>  PBR([[-1]], [[-1, 1]]),
>  PBR([[]], [[-1, 1]])]);
<pbr monoid of degree 1 with 7 generators>
gap> A < B;
Error, no method found! For debugging hints type ?Recovery from NoMethodFound
Error, no 1st choice method found for `<' on 2 arguments at /Users/Wilf/gap/lib/methsel2.g:250 called from
<function "HANDLE_METHOD_NOT_FOUND">( <arguments> )
 called from read-eval loop at *stdin*:53
type 'quit;' to quit to outer loop
brk>
```
Or, more simply:
```
gap> a := Semigroup(RandomPBR(1));
<commutative pbr semigroup of degree 1 with 1 generator>
gap> b := Semigroup(RandomPBR(1));
<commutative pbr semigroup of degree 1 with 1 generator>
gap> a < b;
Error, no method found! For debugging hints type ?Recovery from NoMethodFound
Error, no 1st choice method found for `<' on 2 arguments at /Users/Wilf/gap/lib/methsel2.g:250 called from
<function "HANDLE_METHOD_NOT_FOUND">( <arguments> )
 called from read-eval loop at *stdin*:56
type 'quit;' to quit to outer loop
brk>
```

**Comment by james-d-mitchell** [comment]
Hmmm, is this really a bug? How would you expect to be able to compare such semigroups?

**Comment by james-d-mitchell** [comment]
Good point. Apart from the degree check at the start, the very same method ought to work for all semigroups. @wilfwilson

**Comment by james-d-mitchell** [comment]
Resolved in release of v3.1.0.
